admin-site/src/App.svelte

```diff
@@ -270,8 +295,7 @@
       }).then( response => { 
         if(response.status == 200){
           response.json().then( details => {
-            document.getElementById("openOccasion").style.display = "none";
-            document.getElementById("closeOccasion").style.display = "block";
+            pageState=3;
```

Comment:
I think this function (and openOccasion, delete, etc) need to make sure the events store gets updated. Maybe create a function in `App` like `updateEvents` and use actions to trigger it?

Reply:
Created a getEventsAndStore function that grabs from the server to update the store.